src/Components/Items.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { BsTrash3Fill } from "react-icons/bs";
+import "bootstrap/dist/css/bootstrap.min.css";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import Form from "react-bootstrap/Form";
+export default class Items extends React.Component {
```

Comment:
This component is describing how a single Item looks like. I would recommend naming it in singular because of that. `Item` instead of `Items`